### ISSUE TITLE
Can I Change Image limit text like The limit is 20 Images or video 

### ISSUE THREAD

**Issue by vandanamodi** [Yummypets/YPImagePicker#729]
I want to set my own text for image limit exceed text label

**Describe the solution you'd like**
is there any property that I can change text of Image limit exceed label or if not then can you please update this change in library 
Please check attachment for exact text reference 
![IMG_1327](https://user-images.githubusercontent.com/95464295/154683166-2434326a-1364-4f27-817f-09bbfb0a3af3.PNG)

**Comment by vandanamodi** [comment]
@s4cha  @NikKovIos @kf99916 @gintechsystems  is there any property to change this limit text for library  or can you make it possible to change this please update me for the same?

**Comment by vandanamodi** [comment]
@arielpollack @schveiguy @vietstone-ng @bithavoc @schveiguy @vietstone-ng @vietstone-ng @vietstone-ng @vietstone-ng  please look into this case and update is that possible to change text or not  ?

**Comment by schveiguy** [comment]
Please do not ping random people.

**Comment by gintechsystems** [comment]
Yes, please do not tag random people. @vandanamodi Please look at "warningMaxItemsLimit" which is part of the configuration.wordings (YPImagePickerConfiguration.shared.wordings), this should point you in the right direction. (https://github.com/Yummypets/YPImagePicker/blob/master/Source/Configuration/YPWordings.swift)

**Comment by vandanamodi** [comment]
> **[gintechsystems](/gintechsystems) ** c

ya sorry for inconvenience. i won't. Thank you for the update.

**Comment by NikKovIos** [comment]
I'm closing old issues. Feel free to open new. Or better - make a PR :)
